dnf-behave-tests/features/microdnf/install.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Install package
+
+@bz1691353
+Scenario: Install an RPM without null lines
+  Given I use repository "dnf-ci-fedora"
+   When I execute dnf with args "install glibc"
```

Comment:
Isn't this test supposed to test microdnf?

Reply:
You're right, I'll adjust the test. It is useful also usage of [PR#731](https://github.com/rpm-software-management/ci-dnf-stack/pull/731)